packages/terra-core-docs/src/terra-dev-site/doc/action-header/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,242 @@
+  - Terra provides the ability for actionable components to receive focus in the order they are coded. 
+  - Engineers must ensure that the focus order is expected and preserves page meaning. Code order should follow the logical reading order of the user. Though a TABINDEX attribute should not be required to ensure keyboard use, if used, never use TABINDEX higher than 0.  
+  - Content creators should convey to the engineers the logical reading order of the page. 
+- [2.5.2 Pointer Cancellation](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html) — Supports
+  - Terra components use the up event to perform actions. 
+  - Engineers must ensure actionable controls always take on the up event. 
+- [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html) — Supports 
+  - Terra provides the ability for programmatic labels to match visual labels. 
+  - Engineers must ensure the programmatic label matches the visual label when interactable controls are included within Action Header. Ensure the visible control label is the first word of the programmatic label when associating the control to other words in visual proximity that further describe the control. 
+  - Content creators must provide engineers guidance to ensure unique names for all interactive controls within Terra Action Header.  
+- [3.2.1 On Focus](https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html) — Supports 
+  - Terra components do not cause a change of context on focus by default. 
+  - Engineers must ensure that no unexpected change of context happens when interactive elements receive focus. 
+  - Content creators must use standard, predictable design patterns that support user expectations. 
+- [3.2.4 Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html) — Supports 
+  - Terra provides the ability for elements to be consistently identified. 
+  - Engineers must ensure that elements used for the same function are identified consistently, both visually (same icon for the same functionality) and programmatically (the alternate content behind each image consistently represents the same functionality). 
+  - Content creators must ensure any icons used are used to represent the same function consistently. Additionally, the alternate content used to describe icons must be consistent for users to understand their function predictably. 
+- [4.1.1 Parsing](https://www.w3.org/WAI/WCAG21/Understanding/parsing.html) — Supports 
+  - Terra components are tested and validated before release to ensure proper code parsing. 
+  - Engineers must ensure their code is valid HTML. 
+- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) — Supports 
+  - Terra provides the ability for Name, Role, and Value attributes to be programmatically determined so as to be read by screen readers. 
```

Comment:
Shouldn't these be lowercase? Either way, it'll be prudent to highlight these using monospace:
```suggestion
  - Terra provides the ability for `Name`, `Role`, and `Value` attributes to be programmatically determined so as to be read by screen readers. 
```